### DISCUSSION TITLE
Officially missing monster lore

### DISCUSSION THREAD

**Discussion by SummerSky777** [decheine/complete-compendium#226] [General]
First thing noted, decades ago when i bought the compendium, is that the Iron Golem, gas/poison does not have official errata as it does not state what poison is exhaled by the Golem. 

I suggest some * and a simple very brief note that the type of poison is not defined in the monster entry and no official errata exists (though the spell _Cloudkill_ is used in its construction). 

As the 2e spell required? _Cloudkill_? This is what i had logically assumed was missing.
Computer games version causes some damage per round
As 1e, though its 2e, save or die?
or something else?

https://www.completecompendium.com/appendix/golem1/
"Once every 7 rounds, beginning either the first or second round of combat, the iron golem breathes out a cloud of poisonous gas. It does this automatically, with no regard to the effects it might have. The gas cloud fills a 10 foot cube directly in front of it, which dissipates by the following round, assuming there is somewhere for the gas to go." And it does say what the gas does or any other specifics. 

noted here; https://www.enworld.org/threads/2e-iron-golem-gas-errata.703169/

**Comment by decheine** [comment]
Good observation, I had to double check my own books and indeed there is no description of what the _poisonous gas_ actually does. I have a hunch that the answer could lie hidden in a column in a Dragon or Dungeon magazine, where someone asks this same question and maybe some people from TSR answered. 

The golem I page could probably use a footnote explaining this conundrum and possible options. Personally I don't put this as super high priority but if you find something definitely follow up.

**Reply by decheine** [reply]
this was bugging me so I couldn't help but look into it. My first extra clue we can take is from Dragon # 62, with "Green dragon armor - Resistance against green dragon breath, iron golem breath, _Cloudkill_ spell, and other attack forms using poisonous gases." The issue doesn't mention anything else on iron golems

**Comment by decheine** [comment]
In 9519 Undermountain The Lost Level, in the monster statistics there is an iron golem where it includes
> Notes: SA - The golem can breathe a cloud of poisonous gas once every 7 rounds (save vs. poison or die).

I'm going to look at other references to iron golems so see what else shows up, but that's probably the best we can hope for. In the end I'll make a footnote on the golem1 page

**Reply by decheine** [reply]
but 9220 AC11 The Book of Wondrous Invention describes constructs like iron golems, but 
> For AD&D rules, it does not have the _cloud kill_ attack, nor any of the iron golem's usual immunities and advantages."

So I think the footnote should be something like.

- The original description of the iron golem's poisonous gas attack are unclear, however in TSR 9519 Undermountain The Lost Level the iron golem's ability is described as: a cloud of poisonous gas once every 7 rounds  (save vs. poison or die). Another option is per the 5th level wizard spell _cloudkill_.

**Comment by SummerSky777** [comment]
indeed, this should cover it
